packages/platform/types/event-types/event-types_2024_06_14/outputs/event-type.output.ts

```diff
@@ -423,4 +423,39 @@ export class TeamEventTypeOutput_2024_06_14 extends BaseEventTypeOutput_2024_06_
   @IsOptional()
   @DocsProperty()
   bannerUrl?: string;
+
+  @IsString()
+  @IsOptional()
+  @DocsProperty()
+  teamName?: string;
```

Comment:
The request is to fetch team event type, but we have added properties like `weekStart`, `brandColor` that makes it seem like they reside on the team event type level while in the reality all of the newly added output properties are part of a team, not team event type, and we should not mix them on the same level. We are coupling API with what atom needs too much.

So either:
1. Add a new query parameter to the endpoint returning team event type which is a comma separated string `expand` to which we can pass which team event type properties we want to expand e.g. `?expand=team` and then if it is passed instead return new `team` object in response.
2. Create atoms specific endpoint that returns event type just for the atom in format it needs with all the information it needs. This would also allow us to get ride of the transformApiEventTypeForAtom.ts function.
3. Quickest (albeit suboptimal) solution - define property `team` on the output under which you store all newly added properties and always return them aka it is basically the same as now but creating a nested `team` in this output to not mix event type with team properties.